emperor/support_files/js/emperor.js

```diff
@@ -48,7 +48,9 @@ var g_keyBuilt = false;
 var g_screenshotBind;
 var g_separator_left;
 var g_separator_history;
-
+var g_slider = null;
+var g_taxaLevel = 0;  // taxonomic level to display
```

Comment:
Instead of using this variable to keep track of the selected level, I would retrieve the value directly from the slider.